### DISCUSSION TITLE
Allow the host to set class limits

### DISCUSSION THREAD

**Discussion by Smrtonosan** [FakeFishGames/Barotrauma#15249] [Ideas and suggestions]
Now that medic and Security officer limits have been removed (entirely?) it would be really useful to have an option to place limits on certain classes yourself, so that you don't have 4 medics.

Preferably it would also show what the class limits are in the server lobby.

**Comment by AziKarnage** [comment]
You can technically do this with mods currently, it does seem to be a simple task as well
